frontend/src/components/UserInfoCard.tsx

```diff
@@ -83,22 +83,22 @@ export const UserInfoCard = ({ userInfo }: { userInfo: UserInfo }) => {
     ],
   });
 
-  const emailOnOk = async (newValue: string) => {
+  const openChangePasswordModal = () => {
     dispatch(
       modalsActions.showModal({
-        modalType: ModalTypes.CONFIRM_PASSWORD_MODAL,
+        modalType: ModalTypes.CHANGE_PASSWORD_MODAL,
         modalProps: {
-          actionData: { id: userInfo.id, email: newValue },
+          id: userInfo.id,
         },
       }),
     );
   };
-  const passwordOnOk = async (newValue: string) => {
+  const emailOnOk = async (newValue: string) => {
```

Comment:
Here also the reordering should be undone, since `emailOnOk` has not actually changed at all